site/source/api/index.ejs

```diff
@@ -0,0 +1,10 @@
+---
+description: This is a template used to generate the API index
+title: API Documentation
+---
+<section class="section">
+	<div class="container">
+		<h1 id="API-Documentation"><a href="#API-Documentation" class="headerlink" title="API Documentation"></a>API Documentation</h1>
+		<p><a href="/api/cli/v2.0.0-beta2.3">cli</a><br><a href="/api/compose/v2.0.0-beta1.1">compose</a><br><a href="/api/core/v2.0.0-beta2.4">core</a><br><a href="/api/has/v2.0.0-beta2.3">has</a><br><a href="/api/i18n/v2.0.0-beta2.3">i18n</a><br><a href="/api/interfaces/v2.0.0-beta2.4">interfaces</a><br><a href="/api/loader/v2.0.0-beta2.3">loader</a><br><a href="/api/routing/v2.0.0-beta2.4">routing</a><br><a href="/api/shim/v2.0.0-beta2.4">shim</a><br><a href="/api/stores/v2.0.0-alpha.12">stores</a><br><a href="/api/streams/v2.0.0-alpha.2">streams</a><br><a href="/api/widget-core/v2.0.0-beta2.11">widget-core</a><br><a href="/api/widgets/v2.0.0-beta2.1">widgets</a></p>
+	</div>
+</section>
```

Comment:
Missing empty line at EOF

Reply:
I think this might be coming from the base branch that I forked from @itorrey, but I will look for this in the future.